backend/controllers/usersController.js

```diff
@@ -58,9 +58,9 @@ const registerUser = asyncHandler(async (req, res) => {
                 res.status(400)
                 throw new Error("invalid user data")
             }
-        }else {
+        } else {
             res.status(400)
-            throw new Error("invalid user data") 
+            throw new Error("invalid user data")
         }
     }
 
```

Comment:
here     const userExists = await User.findOne({ email }) 
i tried to find both of email and name using find method but it dosn't work 
So could i make this term or not ?